.github/CODE_STANDARDS.md

```diff
@@ -233,7 +233,7 @@ Additional information and guidelines on Conventional Commits can be found [here
 Good example:
 
 ```bash
```

Comment:
We removed `bash` from here in spv-wallet-web-frontend, so we can do it here as well :)